server/models/thingy.js

```diff
@@ -9,7 +9,6 @@ const type = thinky.type;
 const schema = {
   id: type.string(),
   name: type.string().min(1),
-  price: type.number().min(0.01),
```

Comment:
Did we? I figured we decided based on office manager feedback that price is redundant since it is subject to constant change and can be set only approximately which has no practical use.
That said, I will still wait till you answer and do it as you see fit :+1: